### ISSUE TITLE
DOC: Add Makefile directive to for deploying docs to GitHub pages

### ISSUE THREAD

**Issue by nehalecky** [Melrok/energistream-py#8]
This guy has some thoughts: http://oli.jp/2011/github-pages-workflow/

```bash
cd /path/to/diveintohtml5
$ git checkout -b gh-pages
$ git push origin gh-pages
```

**Comment by hwfross** [comment]
Just for the sake of reference, this is the example structure I was working off of : http://daler.github.io/sphinxdoc-test/includeme.html

**Comment by nehalecky** [comment]
That reference looks like the right way to do it. Please see how far you can get trying to get it up and going. Thanks @hwfross!

**Comment by nehalecky** [comment]
Closed via: 4dc9bd77083161f7403c96f329db8d246ed0c1aa
